x-pack/plugins/observability_solution/apm/ftr_e2e/cypress/e2e/settings/agent_configurations.cy.ts

```diff
@@ -122,21 +122,24 @@ describe('Agent configuration', () => {
       cy.contains('Create configuration').should('not.be.disabled');
     });
 
-    it('persists service environment when clicking on edit button', () => {
+    it.skip('persists service environment when clicking on edit button', () => {
```

Comment:
@cauemarcondes not sure what's going on with this test, but it doesn't seem like `opbeans-node` is an option in the select when the typing takes place. Do you mind taking a look at this one when you have time?

Reply:
Alright done, everything should work as expected now